torch/_export/converter.py

```diff
@@ -458,6 +493,16 @@ def convert_node(self, node: torch._C.Node):
             self.convert_prim_if(node)
         elif node_kind == "aten::Bool":
             self.convert_as_noop(node)
+        elif node_kind == "aten::__is__":
+            self.convert_aten___is__(node)
+        elif node_kind == "aten::__isnot__":
+            self.convert_aten___isnot__(node)
+        elif node_kind == "aten::__not__":
+            self.convert_aten___not__(node)
+        elif node_kind == "profiler::_record_function_enter_new":
+            self.convert_profiler__record_function_enter_new(node)
+        elif node_kind == "profiler::_record_function_exit":
+            self.convert_profiler__record_function_exit(node)
```

Comment:
I wonder if https://github.com/pytorch/pytorch/pull/127580 can generically fix these profiler op issues 
cc @jiashenC

Reply:
It looks like it should do the job, but `exit` does not need a new fx node?